proggy.cpp

```diff
@@ -76,14 +111,20 @@ std::vector<Customer> loadCustomers(std::string fname) {
     std::ifstream f(fname);
     nlohmann::json j;
 
+//** KOOS:
+//** This if statement does nothing - i.e. if (f) was !true, nothing is done about it..
     if (f) {
         j << f;
     }
+ 
+//** KOOS
+//** no try-catch? 
```

Comment:
also fair -- I assumed happy path here else I'd have spent WAY more than an hour on this. :)